packages/sitecore-jss-react/src/index.ts

```diff
@@ -64,6 +65,7 @@ export {
   WithSitecoreContextProps,
   WithSitecoreContextHocProps,
 } from './enhancers/withSitecoreContext';
+export { withErrorBoundary } from './enhancers/withErrorBoundary';
```

Comment:
Wondering if it will be confusing to expose this HOC in the public API surface as we are _automatically_ using it for every component used in a placeholder. Trying to think of a use case other than our own internal usage in `PlaceholderCommon.tsx`. Do we think this will be used on components that are not dynamic?

Reply:
I thought why not make this an option if anyone wants to use it any time in future, but you're probably right - if anyone needs the OOB HOC for this - they'll let us know.